dhall-docs/src/Dhall/Docs.hs

```diff
@@ -42,6 +43,7 @@ data Options
         { packageDir :: FilePath         -- ^ Directory where your package resides
         , docLink :: FilePath            -- ^ Link to the generated documentation
         , resolvePackageName :: Path Abs Dir -> Text
+        , ascii :: Bool
```

Comment:
I believe it's better to use `CharacterSet` here instead of `Bool`. In `Dhall.Main` we already do something similar with `Censor`:

https://github.com/dhall-lang/dhall-haskell/blob/7a00835ea23372b993a14e6be81f5db091f199d7/dhall/src/Dhall/Main.hs#L210-L213

It will be `Unicode` by default, `Ascii` when the `--ascii` flag is present

Reply:
That looks more convenient indeed. Thank you for the suggestion.